ArduPlane/quadplane.cpp

```diff
@@ -741,6 +748,11 @@ bool QuadPlane::setup(void)
     pos_control->set_dt(loop_delta_t);
     attitude_control->parameter_sanity_check();
 
+    // TODO: update this if servo function assignments change
+    // used by relax_attitude_control() to control special behavior for vectored tailsitters
+    _is_vectored = ((motor_class == AP_Motors::MOTOR_FRAME_TAILSITTER) &&
+           (!is_zero(tailsitter.vectored_hover_gain) && SRV_Channels::function_assigned(SRV_Channel::k_tiltMotorLeft)));
```

Comment:
since were only doing this one time you might as well also check tiltMotorRight

Reply:
but you were the one who made me take that check out :)

Reply:
Maybe when it was once a loop? I meant check for one or the other. 
```suggestion
           (!is_zero(tailsitter.vectored_hover_gain) && (SRV_Channels::function_assigned(SRV_Channel::k_tiltMotorLeft) || SRV_Channels::function_assigned(SRV_Channel::k_tiltMotorRIght)));
```

Reply:
I hope I never had that in a loop, but maybe...
What happens if you "commit suggestion". Does it do a weird merge commit? I pushed the "resolve inconsistencies" button recently, and didn't like the result at all.

Reply:
You just get a extra commit on top, the message is abit random usually so I tend to then pull it down squash and push back up. 

Maybe it was on some slower check or something, to be honest I have no idea, maybe I'm just fickle ;)

Reply:
Thanks, I'll give that a try then.